### PR TITLE
Add optional GeoJSON encoding of result data

### LINKED ISSUE #1: return GeoJSON LineStrings
To follow standardised best practices, this library should return the shapes as [GeoJSON `LineString`s](https://tools.ietf.org/html/rfc7946#appendix-A.2) instead of a homegrown array of arrays.

### PR DESCRIPTION
**Pull request by coding46**
Fixes #1

### CODE REVIEW

File: uncompress.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 'use strict'
 
-const uncompress = (input) => {
+const wrapToGeoJson = require('./wrapToGeoJson');
```
Comment (derhuerst) [inline uncompress.js]: Please move this file to a `lib` directory. Don't forget to add it to the `files` field in `package.json`.

File: uncompress.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+
+const uncompress = (input, geoJsonWrap) => {
+	geoJsonWrap = geoJsonWrap || false;
```
Comment (derhuerst) [inline uncompress.js]: Move the default value up into the parameter definition.

File: index.js [hunk 3]
```diff
@@ -8,14 +8,14 @@ const all = require('./ids.json')
 
 
 
-const shape = (id) => new Promise((yay, nay) => {
+const shape = (id, geoJson) => new Promise((yay, nay) => {
```
Comment (derhuerst) [inline index.js]: Please call this parameter `formatAsLineString`.

File: uncompress.js (hunk 1 above)
Comment (coding46) [reply uncompress.js]: Fixed in e1fc305

File: uncompress.js (hunk 2 above)
Comment (coding46) [reply uncompress.js]: Fixed in e1fc305

File: index.js (hunk 3 above)
Comment (coding46) [reply index.js]: Fixed in e1fc305

File: readme.md [hunk 4]
```diff
@@ -49,6 +49,47 @@ allShapes()
 .on('error', console.error)
 ```
 
+### GeoJSON encoded result
+
+By adding the optional param `geoJSON` you'll receive a GeoJSON encoded result set.
```
Comment (derhuerst) [inline readme.md]: Please rename to `formatAsLineString`.

Comment (coding46) [reply readme.md]: Fixed in bf8607f

**Comment by derhuerst** [comment]
Great, thanks!

**Comment by derhuerst** [comment]
Will rebuild and publish `vbb-shapes` on Monday.

**Pull request merged**